src/App/Fossa/Analyze/ScanSummary.hs

```diff
@@ -341,6 +364,15 @@ skippedDueFilter = ": skipped (exclusion filters)"
 skippedDueNonProductionPathFiltering :: Doc AnsiStyle
 skippedDueNonProductionPathFiltering = ": skipped (non-production path filtering)"
 
+skippedReachabilityDueToPartialGraph :: Doc AnsiStyle
+skippedReachabilityDueToPartialGraph = ": skipped (partial graph)"
+
+skippedReachabilityDueToNotSupported :: Doc AnsiStyle
+skippedReachabilityDueToNotSupported = ": skipped (not supported)"
+
+skippedReachabilityDueToMissingAnalysis :: Doc AnsiStyle
+skippedReachabilityDueToMissingAnalysis = ": skipped (no dependency analysis)"
```

Comment:
Consider linking to the doc in comments with a note saying to update the doc if these change.